dangerzone/container/repro-sources-list.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+#
```

Comment:
We also had a discussion about this file, and if it should be included in this repository or not (the other option would be to download it on the fly, which could be done by the `build-image.py` script.).

We didn't came up with a conclusion on this one just yet. It feels odd to have it included here, but might work on the short run.

Reply:
Hm, would a third party notice make things better? Check out 4595b3ad4a834cef307c06d1021069b19502522d